app/components/Button.js

```diff
@@ -42,7 +46,7 @@ const RealButton = styled.button`
   }
 
   ${(props) =>
-    props.neutral &&
+    props.$neutral &&
```

Comment:
what does the leading `$` mean generally?

Reply:
Stops it being added to the dom by styled-components: https://styled-components.com/docs/api#transient-props